onnxscript/function_libs/torch_lib/ops/core.py

```diff
@@ -2063,6 +2063,21 @@ def aten_copy(
     return self
 
 
+@torch_op("aten::_to_copy", trace_only=True)
+def aten__to_copy(
+    self: TTensor,
+    dtype: int = -1,
+    non_blocking: bool = False,  # pylint: disable=unused-argument
+) -> TTensor:
+    """_to_copy(Tensor self, *, ScalarType? dtype=None, Layout? layout=None, Device? device=None, bool? pin_memory=None, bool non_blocking=False, MemoryFormat? memory_format=None) -> Tensor"""
+
+    if dtype == -1:
+        self = op.Identity(self)
```

Comment:
What is the criteria using op.Identity now?

Reply:
In a script fuction, onnx script will add identity for us. In a trace_only function we can add identity when we need to rename a tensor (which came to my mind after I left the previous comment). So in this case, we can add identity since the output of this aten function will take a different name in the fx graph from the input, for readability of the ONNX graph. But we expect this identity to be optimized away in a pass.

Reply:
I updated the suggested code. Thanks!